symbols/arm9/itcm.yml

```diff
@@ -215,3 +231,11 @@ itcm:
         Note: This symbol isn't actually part of the ITCM, it gets created at runtime on the spot in RAM that used to contain the code that was moved to the ITCM.
         
         type: struct mem_block[256]
+    - name: RENDER_3D_FUNCTIONS
+      address:
+        EU: 0x20B3DE0
+        EU-ITCM: 0x1FF8120
+      description: |-
+        Pointers to the 4 function avalaible from render_3d_element (in ITCM)
+        
+        type: void* [4] // array of 4 pointer to functions. Take 1 parameter, a pointer to the render_3d_element
```

Comment:
Function pointers are the one case where this project uses `typedef`. For example, see `move_effect_fn_t` in `dungeon_mode.h`. In this case, instead of trying to describe it in the symbol description, you could define something like

```c
typedef void (*render_3d_fn_t)(struct render_3d_element* element);
```

and then define the global variable like this:

```c
extern render_3d_fn_t RENDER_3D_FUNCTIONS[4];
```